### PR TITLE
Syrup should be real Maple syrup

### PR DESCRIPTION
**Pull request by kdaigle**
This is really important! :+1:

### CODE REVIEW

File: New recipe for French Toast [hunk 1]
```diff
@@ -1,2 +1,4 @@
 #French Toast 
 this is a test of adding a file to a public repo
+
+This is a small delicious syrup based change.
```
Comment (kdaigle) [inline New recipe for French Toast:4]: You should have used a CSS precompiler... duh. :8ball:

**Comment by mike-henderson** [comment]
:pineapple:

**Comment by ytrezq** [comment]
> Hey there,
>
> We prefer users don't disable SSL verification in general and highly
> recommended fixing your SSL certificates to be verifiable. Unfortunately,
> we can't support insecure cipher suites. Our cipher suite choices and TLS
> requirement are an important part of how we keep webhooks secure.
>
> If I can be of any other help, please let me know.
Yes of course, read bounty@github.com for the a phone home case

By the way can you resend my invitation e-mail to the bug bounty hunter organization please ? My primary e-mail address was set to a dead mail server. I forgot to switch to the new one. *(I was paid for a GitHub vulnerability)*

thanks,

**Comment by ytrezq** [comment]
@kdaigle 
> Hey there,
>
> We prefer users don't disable SSL verification in general and highly
> recommended fixing your SSL certificates to be verifiable. Unfortunately,
> we can't support insecure cipher suites. Our cipher suite choices and TLS
> requirement are an important part of how we keep webhooks secure.
>
> If I can be of any other help, please let me know.
Yes of course, read bounty@github.com for the a phone home case

By the way can you resend my invitation e-mail to the bug bounty hunter organization please ? My primary e-mail address was set to a dead mail server. I forgot to switch to the new one. *(I was paid for a GitHub vulnerability)*

thanks,

**Comment by ytrezq** [comment]
I disagree, doing this for breakfirst on a daily basis would implies a 30€ per months increase in food. :laughing:
With 70€ per liter, we tend to not buy maple syrup at all. :smile:

@kdaigle By the way, please check your e‑mails.

**Comment by ytrezq** [comment]
I disagree, doing this for breakfirst on a daily basis would implies a 15€ per months increase in food. :smile:
Generally, we(french) tend to not buy maple syrup at all. :smile:

@kdaigle By the way, please check your e‑mails replies (the above is mainly to stay on topic.).

**Comment by ytrezq** [comment]
I disagree, doing this for breakfirst on a daily basis would implies a 15€ per months increase in food. :smile:
Generally, we(french) tend to not buy maple syrup at all. :smile:

@kdaigle By the way, please check your e‑mails replies _(the above is mainly to stay on topic but you didn’t replied to the question I asked when answering your e‐mail, sorry about that)_.

**Pull request merged**